py/hirm_io.py

```diff
@@ -11,6 +17,30 @@
 DomainCluster = collections.namedtuple(
     "DomainCluster", ["cluster_id", "domain", "entities"])
 
+
+def load_schema(path):
+  """Load the schema from path, and return it as a dict of Relations."""
+  relations = {}
+  comment_line_re = re.compile(r'\s*#.*')
+  # TODO(thomaswc): Handle distribution parameters in brackets
+  line_re = re.compile(r'\s*(\w+)\s*~\s*(\w+)\s*\(\s*(.+)\s*\)\s*(#.*)?')
+  with open(path, 'r') as f:
+    for line in f:
+      if comment_line_re.match(line):
+        next
```

Comment:
This should be continue. Maybe add a test with a comment line?

Reply:
Done and done.  The test revealed another bug (in the regex) which I've also fixed.